### PR TITLE
[velero] Adding possibility to add multiple backup / snapshot locations

### PR DESCRIPTION
**Pull request by fniko**
Added possibility to use multiple backup and snapshot locations.
Based on my previous commit where ENV secrets should be added.

### CODE REVIEW

**Comment by cpanato** [comment]
@fniko can you update your branch and resolve the conflict? thanks!

**Comment by fniko** [comment]
> @fniko can you update your branch and resolve the conflict? thanks!

Fixed!

File: charts/velero/values.yaml [hunk 1]
```diff
@@ -71,45 +71,46 @@ configuration:
...
+      config: {}
+      #  region:
+      #  s3ForcePathStyle:
+      #  s3Url:
+      #  kmsKeyId:
+      #  resourceGroup:
+      #  The ID of the subscription containing the storage account, if different from the cluster’s subscription. (Azure only)
+      #  subscriptionId:
+      #  storageAccount:
+      #  publicUrl:
```
Comment (jenting) [inline charts/velero/values.yaml]: `serviceAccount:` missing?

File: charts/velero/templates/backupstoragelocation.yaml [hunk 2]
```diff
@@ -50,5 +49,5 @@ spec:
   {{- end }}
   {{- end }}
 {{- end }}
-{{- end }}
-{{- end }}
+---
+{{- end}}
```
Comment (jenting) [inline charts/velero/templates/backupstoragelocation.yaml]: nit: `{{- end}}` or `{{- end }}`?

File: charts/velero/templates/volumesnapshotlocation.yaml [hunk 3]
```diff
@@ -34,5 +33,5 @@ spec:
   {{- end}}
 {{- end }}
 {{- end }}
-{{- end }}
-{{- end }}
+---
+{{- end}}
```
Comment (jenting) [inline charts/velero/templates/volumesnapshotlocation.yaml]: nit: `{{- end}}` or `{{- end }}`?

File: charts/velero/templates/deployment.yaml [hunk 4]
```diff
@@ -118,6 +118,13 @@ spec:
               value: {{ default "none" $value }}
           {{- end }}
           {{- end }}
+          {{- range .Values.environmentSecrets }}
```
Comment (carlisia) [inline charts/velero/templates/deployment.yaml:121]: I don't think this is going to work. Velero only works with 1 secret at the time. If I'm misunderstanding, maybe please explain what this is trying to solve, thank you,

**Comment by jenting** [comment]
Ping @fniko, could you please fix the conflicts and the above comments? Thanks.

**Pull request closed without merging**